mappings/net/minecraft/entity/vehicle/AbstractMinecartEntity.mapping

```diff
@@ -14,15 +23,23 @@ CLASS net/minecraft/class_1688 net/minecraft/entity/vehicle/AbstractMinecartEnti
 		ARG 3 x
 		ARG 5 y
 		ARG 7 z
+	METHOD method_18803 willHitBlockAt (Lnet/minecraft/class_2338;)Z
+		ARG 1 pos
+	METHOD method_7504 getMaxSpeedOffRail ()D
```

Comment:
```suggestion
	METHOD method_7504 getMaxOffRailSpeed ()D
```

Reply:
I would actually say "get max speed off the rail" in normal speech, not "get max off rail speed".